pitop/miniscreen/oled/oled.py

```diff
@@ -53,16 +53,20 @@ def __init__(self):
 
     def __setup_subscribe_client(self):
         def on_control_changed(parameters):
-            controller = int(parameters[0])
+            controller = int(parameters()[0])
```

Comment:
I still worry about needing to invoke 'parameters'. Can we change this now before it's all across our codebase?